plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -101,6 +101,21 @@
                 });
             });
         });
+        
+        function boxes_selected(){
+            var checked_boxes = 0;
+            var boxes = document.querySelectorAll('span.checkbox-container > input');
+            
+            for (var i = 0; i < boxes.length; i++){
+                if (boxes[i].checked)
+                    checked_boxes += 1
+            }
+            if (checked_boxes >= 1) {
+                document.getElementById('update-selected-plugins').classList.remove('disabled');
+            } else if (checked_boxes === 0) {
+                document.getElementById('update-selected-plugins').classList.add('disabled');
+            }
+        }
```

Comment:
This function should not be added to the global scope and should not be referenced via `onclick=`. Instead, jQuery should be used to bind an anonymous function to the event (as is done above). Can you make this change?

Also, this code can be simplified to just check if at least one input is checked, eg:

```
var isAtLeastOneChecked = $('span.checkbox-container > input[checked]').length >= 1;
$('#update-selected-plugins').prop('disabled', !isAtLeastOneChecked);
```

Reply:
Hi @diosmosis thanks for your review and your advice
I made a new commit considering your tip :)